### ISSUE TITLE
Developer onboarding: create_test_event fails

### ISSUE THREAD

**Issue by shaib** [pretalx/pretalx#876]
## Current Behavior

Following https://docs.pretalx.org/developer/setup.html, after running the `migrate` and `init` steps, the `create_test_event` fails:
```
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
.
.
.
  File ".../pretalx/.env/lib/python3.7/site-packages/sass.py", line 738, in compile
    raise CompileError(v)
sass.CompileError: Error: File to import not found or unreadable: .../pretalx/src/static.dist/agenda/scss/main.scss.
        on line 3:1 of stdin
>> @import ".../pretalx/src/static.dist/agenda/scss/main.scss";
   ^
```

## Expected Behavior
Either that command should succeed, or another preparatory step should be named in the documentation.

## Workaround (and Possible Fix)
After `init`, run `tox -e dev manage.py collectstatic`.
Then,  `create_test_event` succeeded.

## Environment

* Version used: git master
* Environment name and version: Python 3.7
* Operating System and version (desktop or mobile): Debian (testing)

**Comment by rixx** [comment]
Documentation bug, indeed – thanks for raising the issue.

The issue is due to the custom colour assigned to the test event, which requires compiling custom CSS from SASS. I think I will change the test event to only be generated with custom colours if `DEBUG = False`. Custom event colours mean custom CSS, which is very annoying to debug when working on frontend changes, because we can't have auto updates for them.

**Comment by ThomasWaldmann** [comment]
Following the docs with a current git checkout still leads to the traceback.

**Comment by rixx** [comment]
Running collectstatic will resolve the issue.

**Comment by ThomasWaldmann** [comment]
OK, so it is a documentation bug as this is not mentioned.

https://docs.pretalx.org/developer/setup.html

**Comment by rixx** [comment]
Would've happily taken a PR, which would probably have taken up the same time as this conversation 😉
➡ 3c7cc38a

**Comment by ThomasWaldmann** [comment]
Can confirm, it works after collectstatic.

I also had to do "manage.py reset_db" because otherwise it ran into a unique constraint issue.

**Comment by ThomasWaldmann** [comment]
Well, I don't think I'll work on "prematurely closed" issues.

**Comment by rixx** [comment]
If the issue is still a concern, I'll happily re-open it. Is it?

**Comment by ThomasWaldmann** [comment]
Sure it is as the docs "don't work". This issue is not fixed by the change you have committed.

**Comment by rixx** [comment]
Huh, ok, in that case I misunderstood: running collectstatic was not sufficient for you?

**Comment by ThomasWaldmann** [comment]
It was sufficient. But the docs are incomplete because they do not mention that.

Which was the main issue since this bug was filed I guess. ;-)

**Comment by rixx** [comment]
But the docs do mention it, as of the comment I posted [above](https://github.com/pretalx/pretalx/issues/876#issuecomment-599117658) half an hour ago, referencing 3c7cc38a8c2d36506cb65c93bcf77a5c6ac78c64. I was just confused that you insisted after that change. Closing for good now.

**Comment by ThomasWaldmann** [comment]
As the changeset ID did not link (and I was lazy :), I didn't really see the contents, sorry.

**Comment by ThomasWaldmann** [comment]
Fixed by: 3c7cc38a8c2d36506cb65c93bcf77a5c6ac78c64
